client/src/views/profile/AuthenticationPage.vue

```diff
@@ -68,6 +122,12 @@ async function openChangeAuthentication(): Promise<void> {
     if (answer === Answer.No) {
       return;
     }
+  } else if (currentDevice.value && currentDevice.value.ty.tag === AvailableDeviceTypeTag.OpenBao) {
+    const connResult = await getOpenBaoClient();
+    if (!connResult) {
```

Comment:
Missing some error handling here. What does the user see if we can't get the open bao client?